src/unstract/sdk/file_storage/fs_shared_temporary.py

```diff
@@ -0,0 +1,26 @@
+from unstract.sdk.exceptions import FileStorageError
+from unstract.sdk.file_storage import FileStorage, FileStorageProvider
+
+
+class SharedTemporaryFileStorage(FileStorage):
+    SUPPORTED_FILE_STORAGE_TYPES = [
+        FileStorageProvider.Minio.value,
+        FileStorageProvider.Redis.value,
+    ]
+
+    def __init__(
+        self,
+        provider: FileStorageProvider,
+        storage_config,
```

Comment:
```suggestion
        **storage_config
```
NIT: Might be good to add type hints wherever possible too

Reply:
Done.